usaspending_api/etl/management/commands/load_table_to_delta.py

```diff
@@ -127,6 +127,9 @@ def handle(self, *args, **options):
                 properties=get_jdbc_connection_properties(),
             )
 
+        if table_spec.get("column_names"):
+            df = df.select(table_spec.get("column_names"))
+
```

Comment:
Would be good to add a comment here saying why we're doing this (e.g. to indicate preserving a consistent column ordering)

Reply:
>AWARDS_TYPES on its own makes me think of the different award types (contracts, loans, etc.

Very good point, will make that change and add comment.